src/Microsoft.AspNet.Mvc.Core/Properties/Resources.Designer.cs

```diff
@@ -475,7 +475,7 @@ internal static string FormatAttributeRoute_TokenReplacement_UnescapedBraceInTok
         }
 
         /// <summary>
-        /// Unable to find the required services. Please add all the required services by calling '{0}.{1}' inside the call to '{4}' in the application startup code.
+        /// Unable to find the required services. Please add all the required services by calling '{0}.{1}' inside the call to '{2}' in the application startup code.
```

Comment:
GitHub hides changes here by default. But it looks like the `resx` target hasn't run recently.